src/WebJobs.Script/Host/BlobLeaseManager.cs

```diff
@@ -46,6 +49,8 @@ internal BlobLeaseManager(ICloudBlob lockBlob, TimeSpan leaseTimeout, string hos
             _leaseRetryInterval = TimeSpan.FromSeconds(5);
 
             _timer = new Timer(ProcessLeaseTimerTick, null, TimeSpan.Zero, _leaseRetryInterval);
+
+            _logger = loggerFactory?.CreateLogger("Host.General");
```

Comment:
Please define constants like this in some central place so we can see them together and review for consistency